src/Symfony/Component/Translation/Translator.php

```diff
@@ -214,15 +215,24 @@ public function trans(?string $id, array $parameters = [], string $domain = null
             }
         }
 
+        $message = $catalogue->get($id, $domain);
+
+        $parameters = array_map(function($parameter) use ($locale) {
+            if ($parameter instanceof TranslatableInterface) {
```

Comment:
i tend to like this for a possible contract, hence my :+1: for this approach.

Reply:
@ro0NL I'm not sure about your comment: do I need to do something for it to be contract?